src/common/utils/utils.ts

```diff
@@ -63,7 +63,7 @@ export function getEstimatedFee(): Promise<SatoshiBig> {
       bridgeService.getQueuedPegoutsCount(),
     ])
       .then(([nextPegoutCost, pegoutQueueCount]) => {
-        const estimatedFee = nextPegoutCost / (pegoutQueueCount + 1);
+        const estimatedFee = pegoutQueueCount > 0 ? nextPegoutCost / pegoutQueueCount : 0;
```

Comment:
We had changed the calculation because of this: https://github.com/rsksmart/RSKIPs/blob/5d77c46ac78ab85082cf266f2e05b13fba3362d8/IPs/RSKIP385.md#specification